### DISCUSSION TITLE
Vector tiles support?

### DISCUSSION THREAD

**Discussion by Psijic** [osmdroid/osmdroid#1923] [Ideas]
Hello, I did see this topic somewhere but vector maps are the modern reality, do you plan to support it? Currently mapsforge/vtm support it, OsmAnd have an abandoned SDK, Maps.Me has open code and I saw other apps you could cooperate with. The performance of vector maps rendering is good even in OsmAnd where max detalization is used.

**Comment by spyhunter99** [comment]
we have an adapter for mapsforge but beyond that, it's unlikely unless a contributor adds a PR for an adapter for it. Lots of existing issues regarding this
